core/src/tools/testfind_fd.cc

```diff
@@ -67,7 +69,7 @@ void ProcessFileset(directordaemon::FilesetResource* director_fileset,
   me->compatible = true;
 
   if (!CheckResources()) {
-    printf("Problem checking resources!");
+    std::cout << "Problem checking resources!";
```

Comment:
Is not printing '\n' at the end here intentional ?

Reply:
unintentional